docs/4-user-guide/1-the-zarf-cli/100-cli-commands/zarf_package_uninstall.md

```diff
@@ -0,0 +1,27 @@
+## zarf package uninstall
```

Comment:
Although "uninstall" is an accurate verb, I think it will confuse things since we don't use "install" for the most part (or at least try not to).  I wonder if there's an alternate word?

- remove
- rollback
- destroy
- purge
- annihilate
- rob
- erase
- terminate

https://www.wordhippo.com/what-is/the-opposite-of/install.html
wordhippo.com/what-is/the-opposite-of/deploy.html

Reply:
I'll go with remove for now